tests/openedx_learning/apps/authoring/publishing/test_api.py

```diff
@@ -1075,3 +1075,66 @@ def test_multiple_layers_of_containers(self):
         assert unit_change.affected_by.first().cause == component_change
         assert subsection_change.affected_by.count() == 1
         assert subsection_change.affected_by.first().cause == unit_change
+
+
+class GeneralTestCase(TestCase):
+    """
+    Test general operations with the publishing API.
+    """
+    now: datetime
+    learning_package_1: LearningPackage
+    learning_package_2: LearningPackage
+
+    @classmethod
+    def setUpTestData(cls) -> None:
+        cls.now = datetime(2024, 1, 28, 16, 45, 30, tzinfo=timezone.utc)
```

Comment:
Nit: This isn't important, but I generally try to use a start date that reflects when this test was added.